lambdas/data-migration2/tests/test-granulesAndFiles.js

```diff
@@ -127,9 +127,22 @@ test.beforeEach(async (t) => {
     t.context.knex,
     testExecution
   );
+
   t.context.executionCumulusId = pgExecution.cumulus_id;
   t.context.testExecution = testExecution;
 
+  const completedTestExecution = fakeExecutionRecordFactory({
+    status: 'completed',
```

Comment:
This should automatically be set to completed, right? Since that is the default? https://github.com/nasa/cumulus/pull/2896/files#diff-e4b95ee0e3b92433a8e670895c1e8da32d083c3ca64e31761d4bb353cbb791a7R137

Reply:
Hmm, something isn't right there.   Looking.  👀

Reply:
It's not the default though, this is the *database* module factory, not the api/lib one.

Reply:
Ah 🤦‍♀️ I got confused. Sorry about that